code/modules/events/_event.dm

```diff
@@ -97,7 +97,7 @@
 	if(random)
 		log_game("Random Event triggering: [name] ([typepath])")
 	if (alert_observers)
-		deadchat_broadcast(" has just been[random ? " randomly" : ""] triggered!", "<b>[name]</b>") //STOP ASSUMING IT'S BADMINS!
+		deadchat_broadcast(" has just been randomly triggered!", "<b>[name]</b>") //STOP ASSUMING IT'S BADMINS!
```

Comment:
```suggestion
		deadchat_broadcast(" has just been triggered!", "<b>[name]</b>") //STOP ASSUMING IT'S BADMINS!
```
If an event it triggered by an admin it is not accurate.
Better to just say it has been triggered instead of spreading misinformation IMO.

Reply:
Also, you can remove the comment as the "stop assuming it's badmins" it no longer valid.
Now all of them will be assumed to be badmins.

Reply:
yeah good idea